criu/mem.c

```diff
@@ -172,17 +175,58 @@ static int generate_iovs(struct pstree_item *item, struct vma_area *vma, struct
 	unsigned long pages[3] = {};
 	int ret = 0;
 
+	static char *ZERO_PAGE = NULL;
+	static char *REMOTE_PAGE = NULL;
```

Comment:
We tend to follow the [kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html) where variables are usually not capitalized.

Reply:
Got it. Fixed.